WormholyTests/WormholyTests.swift

```diff
@@ -0,0 +1,169 @@
+            )
+        )
+
+        let expected = "curl -H 'accept: application/json' -H 'accept-language: en-US' -H 'authorization: Bearer \(WormholyTests.authenticatedToken)' -H 'content-type: application/json' -H 'host: app-uk.dev.babylontech.co.uk' -H 'x-app-name: babylon' -H 'x-app-version: 4.7.0' -H 'x-platform: ios' -H 'x-platform-version: 12.4' --data-binary '{\"name\":\"Erich Zann\"}' -X PUT --compressed 'https://app-uk.dev.babylontech.co.uk/api/v1/patients/9541561'"
+
+        XCTAssertTrue(generated == expected)
+    }
+
+    func test_curl_patch() {
+        let generated = RequestModelBeautifier.curlExport(
+            request: RequestModel.fixture(
+                endpoint: .patientDetails,
+                method: .patch,
+                token: WormholyTests.authenticatedToken,
+                httpBody: try? JSONSerialization.data(withJSONObject: WormholyTests.payload, options: [])
+            )
+        )
+
+        let expected = "curl -H 'accept: application/json' -H 'accept-language: en-US' -H 'authorization: Bearer \(WormholyTests.authenticatedToken)' -H 'content-type: application/json' -H 'host: app-uk.dev.babylontech.co.uk' -H 'x-app-name: babylon' -H 'x-app-version: 4.7.0' -H 'x-platform: ios' -H 'x-platform-version: 12.4' --data-binary '{\"name\":\"Erich Zann\"}' -X PATCH --compressed 'https://app-uk.dev.babylontech.co.uk/api/v1/patients/9541561'"
+
+        XCTAssertTrue(generated == expected)
+    }
+
+    func test_curl_delete() {
+        let generated = RequestModelBeautifier.curlExport(
+            request: RequestModel.fixture(endpoint: .regions, method: .delete, token: WormholyTests.authenticatedToken)
+        )
+
+        let expected = "curl -H 'accept: application/json' -H 'accept-language: en-US' -H 'authorization: Bearer \(WormholyTests.authenticatedToken)' -H 'content-type: application/json' -H 'host: app-uk.preprod.babylontech.co.uk' -H 'x-app-name: babylon' -H 'x-app-version: 4.7.0' -H 'x-platform: ios' -H 'x-platform-version: 12.4' -X DELETE --compressed 'https://app-uk.preprod.babylontech.co.uk/api/v1/regions'"
+
+        XCTAssertTrue(generated == expected)
+    }
+}
+
+extension WormholyTests {
+    fileprivate enum HTTPMethod: String {
+        case get
+        case post
+        case put
+        case patch
+        case delete
+
+        var value: String {
+            return self.rawValue.uppercased()
+        }
+    }
+
+    fileprivate enum Endpoints: String {
+        case authentication = "https://services-uk.preprod.babylontech.co.uk/ai-auth/v1/authenticate"
+        case memberConditions = "https://services-uk.preprod.babylontech.co.uk/cm-bff/v1/member_conditions"
+        case regions = "https://app-uk.preprod.babylontech.co.uk/api/v1/regions"
+        case patientDetails = "https://app-uk.dev.babylontech.co.uk/api/v1/patients/9541561"
+        case createAccount = "https://app-uk.dev.babylontech.co.uk/api/v2/patients"
```

Comment:
I think we shouldn't put those URLs here. This is a public fork and we have hidden URLs in the main app for some reason.

I think it's better to now have them here, as there are no benefits (it doesn't matter from test purpose if you test with babylon's URLs or Google's URL)